temporian/implementation/numpy/operators/base.py

```diff
@@ -37,6 +38,15 @@ def __call__(self, **inputs: EventSet) -> Dict[str, EventSet]:
     def output_schema(self, key: str) -> Schema:
         return self._operator.outputs[key].schema
 
+    def apply_feature_wise(
+        self,
+        src_timestamps: np.ndarray,
+        src_feature: np.ndarray,
+    ) -> np.ndarray:
+        """If implemented, the op can be executed feature wise."""

```

Comment:
Executes the op on a single feature. Some operators might not implement this.

Reply:
*Done (update to keep under 80 chars).